### PR TITLE
works now

### PR DESCRIPTION
**Pull request by JasonJPeng**
The html CSS works now

### CODE REVIEW

**Review by zanethomas — CHANGES_REQUESTED** [review]
I think it will be better if you (and Keiko) coordinate adding the search result markup as she has prototyped it in her html. She is using rows and three columns.

File: index.html [hunk 1]
```diff
@@ -68,9 +68,28 @@
         <h2>Top Artist</h2>
         <ul id="topList"></ul>
     </div>
-    <div id="image-view">Image View</div>
+   
 
     <div class="container">
+     <!-- Added by Jason    -->
```
Comment (zanethomas) [inline index.html]: I'm curious to know why you added this instead of going with Keiko's design.

**Review by zanethomas — CHANGES_REQUESTED** [review]
Please remove all files which are not part of the functioning app; they should not be merged into master.

**Review by zanethomas — CHANGES_REQUESTED** [review]
Don't use the master branch as a way to backup files. The only things that should be in the master branch are those things which contribute to a working site.

**Review by zanethomas — APPROVED** [review]

**Review by zanethomas — APPROVED** [review]

**Review by vikakeiko — APPROVED** [review]

**Pull request merged**